RecoEgamma/EgammaElectronProducers/python/gsfElectrons_cfi.py

```diff
@@ -345,4 +345,10 @@
                                  ),
 )
 
+ecalDrivenGsfElectronsFromMultiCl = ecalDrivenGsfElectrons.clone()
 
+from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
+phase2_hgcal.toModify(
```

Comment:
no need for era modifier and also better be in a cff file e.g. gsfElectronModules_cff
or even altogether in gsfElectronSequence_cff if nothing else pulls these FromMultiCl modules in from in-between